### DISCUSSION TITLE
Image resolution in Github Markdown

### DISCUSSION THREAD

**Discussion by krondorl** [community/community#42424] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello.
Which image size is recommended for Github Markdown for the image to look sharp on desktop and mobile?
Thanks

**Comment by samharp** [comment]
There is no hard-fast rule to what your image sizes should be, it just comes down to what the image is for and the context it is in. In the case of GitHub Readmes, there are no resolution recommendations given directly by GitHub, so it comes down to how big you would like your image to be in the given area. The area for a readme doesn't look like it spans wider than 894px on desktop, so anything larger than that would be overkill (even for a full-width image).

To ensure the image looks sharp on both desktop and mobile, you could opt for including the image as an SVG? I'm not sure what kind of image you are using so this may not be an option, but this is by far the best way to ensure it looks crisp on every screen size.

I hope this helps!

**Reply by krondorl** [reply]
Thanks, ok!
